config/locales/doc_auth/en.yml

```diff
@@ -190,12 +191,11 @@ en:
         state-issued ID. Then, you’ll take a photo of yourself to confirm that
         you are the owner of the ID.
       upload_no_image_storage: We do not store images you upload. We only verify your identity.
-      verify_identity: We’ll ask for your personal information. We’ll use, keep and
-        share some of your personal information to verify your identity against
-        public records.
+      verify_identity: We'll ask for your personal information to verify your identity
```

Comment:
I wonder why `make normalize_yaml` isn't enforcing this as smart quotes like existed before `’`. 

I can follow-up to see if it's a bug with the normalization.

Reply:
Follow-up fix at: #5621